lib/bootloader_pvm.pm

```diff
@@ -178,14 +178,17 @@ sub prepare_pvm_installation {
     if (is_agama) {
         record_info("Installing", "Please check the expected product is being installed");
         assert_screen('agama-installer-live-root', 400);
-        return;
+    }
+    else {
+        assert_screen("run-yast-ssh", 300);
     }
 
-    assert_screen("run-yast-ssh", 300);
-
-    if (!is_upgrade && !get_var('KEEP_DISKS')) {
+    # For Agama unattended test, in profile it will format disk by default. 
```

Comment:
s/For Agama unattended test, in profile it will format disk by default. /For Agama unattended tests, disks will be formatted by default/
(no dot at the end of comment :) )

Reply:
Ok, updated.

Reply:
Based on my test, we may need some pre scripts for unattended install, see https://suse.slack.com/archives/C082VE1U2F5/p1735877623523389

My draft PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20898